tests/ssl-opt.sh

```diff
@@ -375,9 +375,10 @@ requires_not_i686() {
 
 # Calculate the input & output maximum content lengths set in the config
 MAX_CONTENT_LEN=16384
-MAX_IN_LEN=$( ../scripts/config.py get MBEDTLS_SSL_IN_CONTENT_LEN || echo "$MAX_CONTENT_LEN")
-MAX_OUT_LEN=$( ../scripts/config.py get MBEDTLS_SSL_OUT_CONTENT_LEN || echo "$MAX_CONTENT_LEN")
+MAX_IN_LEN=$( get_config_value_or_default "MBEDTLS_SSL_IN_CONTENT_LEN" )
```

Comment:
I felt it was impossibly unlikely to happen, since a default value is defined in `include/mbedtls/ssh.h` in case no config option is defined. Therefore, the only time this would unexpectedly fail is if a developer removed the defaults in `ssh.h` by accident, in which case MbedTLS would not compile to begin with. What do you think?

Reply:
Yes, you're right. Leave it as is :)

Reply:
Right. This problem can happen with `config.py get` (which only looks at `config.h`) but not with `query_compile_time_config` or `get_config_value_or_default` (which use the code in `query_config.c`, which includes all headers.).